File: ex0412-fizzBuzz/ex0412.py
class Solution:
    def fizzBuzz(self, n: int):
        strArr = list();

        for i in range(n):
            if (i+1)%3 == 0 and (i+1)%5 == 0:
                strArr.append("FizzBuzz")
            
            elif (i+1)%3 == 0:
                strArr.append("Fizz")
            
            elif (i+1)%5 == 0:
                strArr.append("Buzz")

            else:
                strArr.append(str(i+1))

        return strArr

File: ex0412-fizzBuzz/ex0412.c
/**
* Note: The returned array must be malloced, assume caller calls free().
*/
char ** fizzBuzz(int n, int* returnSize){
    int i, j;

    // Alocar memória da matriz
    char ** strArr = (char **) malloc(n*sizeof(char *));
    for(i=0; i<n; i++) {
        strArr[i] = (char *) malloc(9*sizeof(char));
    }

    for(i=0; i<n; i++) {
        if ((i+1)%3 == 0 && (i+1)%5 == 0) {
            strArr[i] = "FizzBuzz";

        } else if ((i+1)%3 == 0) {
            strArr[i] = "Fizz";
        
        } else if ((i+1)%5 == 0) {
            strArr[i] = "Buzz";
        
        } else {
            sprintf(strArr[i], "%d", (i+1));
        }
    }
    
    *returnSize = n;
    return strArr;
}